From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Thu, 13 Oct 2022 12:03:31 +0200
Subject: fpi-image: Check for PIXMAN presency using #ifdef

That's a defined variable that may be there or not, and currently we
warn with:
 - fpi-image.c:29:5: warning: "HAVE_PIXMAN" is not defined, evaluates to 0
---
 libfprint/fpi-image.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libfprint/fpi-image.c b/libfprint/fpi-image.c
index 47aac8d..b21982e 100644
--- a/libfprint/fpi-image.c
+++ b/libfprint/fpi-image.c
@@ -25,7 +25,7 @@
 
 #include <nbis.h>
 
-#if HAVE_PIXMAN
+#ifdef HAVE_PIXMAN
 #include <pixman.h>
 #endif
 
@@ -107,7 +107,7 @@ fpi_mean_sq_diff_norm (const guint8 *buf1,
   return res / size;
 }
 
-#if HAVE_PIXMAN
+#ifdef HAVE_PIXMAN
 FpImage *
 fpi_image_resize (FpImage *orig_img,
                   guint    w_factor,
